build.ps1

```diff
@@ -262,6 +262,17 @@ function cleanExtension([string] $bitness) {
             Remove-Item -Recurse -Force $_.FullName
         }
     }
+
+    $removeRuntimes = @('linux', 'osx', 'win7', 'win8', 'win81')
```

Comment:
Wouldn't it be better to whitelist the runtimes to keep and remove everything else?